buildfix.sh

```diff
@@ -14,9 +63,52 @@ gofmt -w .
 
 if which clang-format &>/dev/null; then
   echo "Formatting .proto files..."
-  clang-format -i --style=Google $(git ls-files --exclude-standard | grep '\.proto$')
+  git ls-files --exclude-standard | grep '\.proto$' | xargs -d '\n' --no-run-if-empty clang-format -i --style=Google
 else
-  echo -e "${c_yellow}WARNING: Missing clang-format; will not format proto files.${c_reset}"
+  echo -e "${c_yellow}WARNING: Missing clang-format tool; will not format proto files.${c_reset}"
 fi
 
-echo "All Done!"
+echo "Formatting frontend and markup files..."
+bazel run --ui_event_filters=-info,-stdout,-stderr --noshow_progress //tools/prettier:fix
+
+if ((gazelle)); then
+  echo "Fixing BUILD dependencies with gazelle..."
+  bazel run --ui_event_filters=-info,-stdout,-stderr --noshow_progress //:gazelle
+fi
```

Comment:
One reason I like the current buildfix script is that it's very fast to run. Using bazel to run tools like this is typically not that fast. Could we look for a version of these tools (prettier, gazelle) on the system and use them if present, otherwise fall back to bazel run?

That will let folks build gazelle / prettier once and throw them in /usr/local/bin and then this command should be instantaneous again

Reply:
SG, done